validator/engine/validator.js

```diff
@@ -5932,6 +5932,10 @@ amp.validator.ValidationResult.prototype.outputToTerminal = function(
   const {status} = this;
   if (status === amp.validator.ValidationResult.Status.PASS) {
     terminal.info('AMP validation successful.');
+    terminal.info('Review our \'Publishing checklist\' to ensure '
```

Comment:
not sure why `Publishing` is capitalized

Reply:
Hmm, yeah, sorry, it should be lower case.